spring-xsuaa-it/src/test/java/testservice/api/XsuaaJwtDecoderTest.java

```diff
@@ -30,6 +31,7 @@
 @SpringBootTest(classes = { SecurityConfiguration.class, MyEventHandler.class,
 		XsuaaAutoConfiguration.class,
 		XsuaaResourceServerJwkAutoConfiguration.class })
+@EnableConfigurationProperties(value = SecurityConfiguration.class)
```

Comment:
We need to get rid of that as well, otherwise we need to inform customers about this incompatible change, in case they define custom security configurations in their productive or testing environment